test/mason/pkgconfig-tests/SKIPIF

```diff
@@ -1,22 +1,41 @@
 #!/bin/bash
 
 
-skip="False"
+skipZlib="False"
```

Comment:
While we're here, it might make this `SKIPIF` a lot more understandable if we renamed these variables something like:

`has_zlib` or `zlib_available`